terraform/paas/resources.tf

```diff
@@ -0,0 +1,104 @@
+resource "cloudfoundry_app" "find-teacher-training" {
+  for_each =  var.app
```

Comment:
As discussed, we will only deploy one app (find) to one environment in each terraform run. Therefore we can simplify and avoid the for_each loop.